yb-voyager/cmd/exportData.go

```diff
@@ -276,6 +276,14 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 	for _, table := range tableList {
 		dbzmTableList = append(dbzmTableList, table.Qualified.Unquoted)
 	}
+	if exporterRole == SOURCE_DB_EXPORTER_ROLE && changeStreamingIsEnabled(exportType) {
+		err := UpdateMigrationStatusRecord(func(record *MigrationStatusRecord) {
+			record.TableListExportedFromSource = dbzmTableList
+		})
```

Comment:
Empty tables are not included in the `dbzmTableList` -- they are already filtered by the caller before calling this function.

Reply:
Already handled in a previous PR - wherein empty tables are not filtered out for live migration.